index.js

```diff
@@ -37,15 +37,21 @@ module.exports = {
     };
 
     await appBlueprint.install(appOptions);
-    await Promise.all([
+
+    let tasks = [
       this.updateTestAppPackageJson(path.join(testAppPath, 'package.json')),
       this.overrideTestAppFiles(
         testAppPath,
         path.join(options.target, 'test-app-overrides')
       ),
       fs.unlink(path.join(testAppPath, '.travis.yml')),
-      this.setupReleaseIt(options.target),
-    ]);
+    ];
+
+    if (options.releaseIt) {
```

Comment:
Good to know that this is working. I thought we need to declare the available options beforehand, which happens [here in ember-cli](https://github.com/ember-cli/ember-cli/blob/master/lib/commands/addon.js#L9-L29), but that's out of our control. 

I guess the only downside here is that this option won't be announced for `ember help addon`! But that's ok for now. Should we backport this blueprint into the official ember-cli repo, then we can add those changes.

Reply:
I think it's likely a fault of ember-cli if a blueprint can't provide its own flags.

I know that the component blueprint in ember-source can provide its own flags. -- maybe I'll look in to that in a followup PR as there are a number of flags I'm wanting to add to this blueprint.